core/state/node/schema.go

```diff
@@ -44,12 +44,12 @@ const AppStateInstalled = "installed"
 const AppStateUninstalled = "uninstalled"
 
 type AppInstallationState struct {
-	*AppInstallation
-	State string `json:"state"`
+	*AppInstallation `tstype:",extends,required"`
```

Comment:
sorry, i've never used `tstype` -- is there documentation you can point me to?

Reply:
You can check out the repo: https://github.com/gzuidhof/tygo

This annotation was so it could handle the embedded struct correctly.